client/src/app/wormhole/client_worker.ts

```diff
@@ -164,7 +164,7 @@ export default class ClientWorker implements ClientInterface {
     id,
     error,
   }: RPCMessage): Promise<void> {
-    window.history.pushState({}, "", "/#/s?cancel=");
+    window.history.pushState({}, "", "/s?cancel=");
```

Comment:
Also If I upload file and cancel, it redirect to receiver:
<img width="701" alt="image" src="https://user-images.githubusercontent.com/104410044/189674392-8ba71315-8331-4943-a912-5e2e1d93a734.png">